### DISCUSSION TITLE
"Motrix" cannot take over the browser downloads.

### DISCUSSION THREAD

**Discussion by ranslong** [agalwood/Motrix#1461] [Q&A]
After installing the Motrix WebExtension on Google Chrome browser, it cannot take over the browser downloads.

**Comment by CW-B-W** [comment]
I think this issue should be posted on [gautamkrishnar/motrix-webextension](https://github.com/gautamkrishnar/motrix-webextension/issues) instead of here.

Did you correctly setup the extension with [motrix-webextension#how-to-use](https://github.com/gautamkrishnar/motrix-webextension#how-to-use)?

`motrix-webextension` has some known issues and I've patched them with [CW-B-W/motrix-webextension](https://github.com/CW-B-W/motrix-webextension/releases/tag/temporary). You may try it and see if problem still exists.
